plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieCatalogModule.java

```diff
@@ -48,6 +48,7 @@ public static NessieIcebergClient createNessieIcebergClient(IcebergNessieCatalog
         return new NessieIcebergClient(
                 HttpClientBuilder.builder()
                         .withUri(icebergNessieCatalogConfig.getServerUri())
+                        .withEnableApiCompatibilityCheck(false)
```

Comment:
Ack: we can enable it after 0.60.0 because we have "Defer API compatibility check until first request"

https://github.com/projectnessie/nessie/pull/6893